src/index.js

```diff
@@ -77,11 +82,7 @@ const mapToPropertyDescriptor = (
         return this.hasAttribute(name);
       },
       set(value) {
-        if (value) {
-          this.setAttribute(name, '');
-        } else {
-          this.removeAttribute(name);
-        }
+        setBooleanAttribute(this, name, value);
```

Comment:
are we sure we want to mix pure and impure functions in the utils file? just thinking do you think we should have a separate file for impure stuff or leave it in this index file. Or do you think mixing is not a problem?